bios_windows.go

```diff
@@ -0,0 +1,29 @@
+// Use and distribution licensed under the Apache license version 2.
+//
+// See the COPYING file in the root project directory for full text.
+//
+
+package ghw
+
+import "github.com/StackExchange/wmi"
+
+type CIM_BIOSElement struct {
```

Comment:
Same comment about naming of the struct. Please use `biosElement` and just pass a const string of `SELECT InstallDate, Manufacturer, Version FROM CMI_BIOSElement` to `cmi.Query()` below.